src/heifimage_int.hpp

```diff
@@ -0,0 +1,94 @@
+// ***************************************************************** -*- C++ -*-
+/*
+ * Copyright (C) 2004-2019 Exiv2 authors
+ * This program is part of the Exiv2 distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, 5th Floor, Boston, MA 02110-1301 USA.
+ */
+/*!
+  @file    heifimage_int.hpp
+  @brief   Include HEIF header files.
+  @author  Peter Kovář (1div0)
+           <a href="mailto:peter.kovar@reflexion.tv">peter.kovar@reflexion.tv</a>
+  @date    25-Dec-18, 1div0: created
+ */
+#pragma once
+
+// *****************************************************************************
+
+// included header files
+#include "image_int.hpp"
```

Comment:
Other image headers such as `jp2image.hpp` include the header `image.hpp` where the class `Image` is defined. When opening this file in an IDE, I get lot of errors from the clang static analyser regarding files not found.

Furthermore, It looks like this file should be named `heifimage.hpp` without the `_int`suffix , and be moved to the folder `include/exiv2` in order to be deployed together with the other supported file format headers.